### PR TITLE
Update branding and screenshot

### PR DESCRIPTION
**Pull request by DFoxinator**

### CODE REVIEW

**Comment by DFoxinator** [comment]
@weikinhuang please check whenever. I still want to probably improve the description.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "chrome-xhr-inspector",
-  "description": "Adds Firebug like XHR console view details to Chrome",
+  "name": "Chromonocle",
+  "description": "Chromonocle is an easy-to-use XHR (ajax) request inspector designed specifically for Chrome. It seamlessly brings the convenient XHR request inspecting functionality found in Firebug to the Chrome console. Chromonocle allows developers to see all the important info about ajax requests without ever leaving the console tab.",
   "authors": [
     {
       "name": "Wei Kin Huang"
```
Comment (weikinhuang) [inline package.json:6]: add yourself to authors

File: bower.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Chrome XHR Inspector",
-  "description": "Adds Firebug like XHR console view details to Chrome",
+  "name": "Chromonocle",
+  "description": "Chromonocle is an easy-to-use XHR (ajax) request inspector designed specifically for Chrome. It seamlessly brings the convenient XHR request inspecting functionality found in Firebug to the Chrome console. Chromonocle allows developers to see all the important info about ajax requests without ever leaving the console tab.",
   "authors": [
     {
       "name": "Wei Kin Huang"
```
Comment (weikinhuang) [inline bower.json:6]: add yourself to authors

Comment (DFoxinator) [inline bower.json:6]: Thanks, added!

**Comment by weikinhuang** [comment]
+1

**Pull request merged**